### ISSUE TITLE
How to Fix ERROR: Could not find a version that satisfies the requirement opencv-python==3.4.2.17 (from manimlib) (from versions: 3.4.8.29, 3.4.9.31, 3.4.9.33, 4.1.2.30, 4.2.0.32, 4.2.0.34) ERROR: No matching distribution found for opencv-python==3.4.2.17 (from manimlib)

### ISSUE THREAD

**Issue by yutharsan** [3b1b/manim#960]
### If this is a support request:

**Please attempt to solve the problem on your own before opening an issue.**
Between old issues, StackOverflow, and Google, you should be able to find
solutions to most of the common problems.

Include at least:
1. Steps to reproduce the issue (e.g. the command you ran)
2. The unexpected behavior that occurred (e.g. error messages or screenshots)
3. The environment (e.g. operating system and version of manim)

### If this is a feature request:
Include the motivation for making this change.

**Comment by einarf** [comment]
You can manually upgrade each pacakge to the latest version. I'm guessing you are using python 3.8 and will need newver versions for it to work.

**Comment by architsangal** [comment]
I tried the following commands:-
a) To install the latest version of opencv-python: - python3 -m pip install opencv-python
   Then, I got this warning.

WARNING: The scripts f2py, f2py3 and f2py3.8 are installed in '/home/architsangal/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.

b) After adding to the path, warning didn't come.

c) Again, I tried to run this command - python3 -m pip install manimlib
  But still getting the same error.

Using Python 3.8.2
